__DEFINES/reagents.dm

```diff
@@ -568,7 +568,8 @@ var/list/cheartstopper = list(/*"potassium_chloride",*/ CHEESYGLOOP) //this stop
 #define BICARIDINES list(BICARIDINE, OPIUM)
 #define SPACE_DRUGGS list(SPACE_DRUGS, MESCALINE)
 #define SYNAPTIZINES list(SYNAPTIZINE, CYTISINE)
-#define HYPERZINES list(HYPERZINE, COCAINE, METHAMPHETAMINE)
+#define HYPERZINES list(HYPERZINE, COCAINE, LIQUIDPCP, METHAMPHETAMINE) //all hyperzine calls call this, except a couple recipes which call safehyperzines
+#define SAFEHYPERZINES list(HYPERZINE, COCAINE) //here so some recipes don't get fucked up and turn recursive
```

Comment:
Cocaine being a "safe" hyperzine?

Reply:
it's just the name of the define, to not have PCP and meth be called on some of the recipe checks for HYPERZINES, and cocaine is mechanically the same as hyperzine